azclishell/app.py

```diff
@@ -419,7 +419,12 @@ def _special_cases(self, text, cmd, outside):
                 telemetry.track_ssg('outside', cmd)
 
             elif text[0] == SELECT_SYMBOL['exit_code']:
-                print(self.last_exit)
+                if self.last_exit == 0:
+                    meaning = 'Success'
+                else:
+                    meaning = 'Failure'
```

Comment:
Good candidate for a ternary operator here
http://book.pythontips.com/en/latest/ternary_operators.html#ternary-operators